### ISSUE TITLE
Model training fails to update status

### ISSUE THREAD

**Issue by alexandreLamarre** [rancher/opni#1825]
Workaround : check `opni-svc-gpu-controller` pod to make sure the model training is occurring / progressing

e.g. : 
```
2023-11-03 17:40:22,655 - INFO - #######Training Model within 3 epochs...######
2023-11-03 17:40:23,177 - INFO - | Epoch: 0 | Total Progress: 0.07% | Training Time Taken: 0.52s | ETC: 704.48s | Epoch Step: 1/450 | Loss: 8.7368 | Tokens per Sec: 2300.56 |
2023-11-03 17:40:49,980 - INFO - | Epoch: 0 | Total Progress: 7.48% | Training Time Taken: 27.33s | ETC: 337.67s | Epoch Step: 101/450 | Loss: 2.8056 | Tokens per Sec: 2273.43 |
2023-11-03 17:41:17,231 - INFO - | Epoch: 0 | Total Progress: 14.89% | Training Time Taken: 54.58s | ETC: 311.42s | Epoch Step: 201/450 | Loss: 0.9982 | Tokens per Sec: 2227.26 |
2023-11-03 17:41:44,152 - INFO - | Epoch: 0 | Total Progress: 22.30% | Training Time Taken: 81.50s | ETC: 283.50s | Epoch Step: 301/450 | Loss: 0.8353 | Tokens per Sec: 2251.41 |
2023-11-03 17:42:11,298 - INFO - | Epoch: 0 | Total Progress: 29.70% | Training Time Taken: 108.64s | ETC: 256.36s | Epoch Step: 401/450 | Loss: 0.7155 | Tokens per Sec: 2229.75 |
2023-11-03 17:42:24,876 - INFO - | Epoch: 1 | Total Progress: 33.41% | Training Time Taken: 122.22s | ETC: 242.78s | Epoch Step: 1/450 | Loss: 0.6694 | Tokens per Sec: 2316.45 |
2023-11-03 17:42:51,903 - INFO - | Epoch: 1 | Total Progress: 40.81% | Training Time Taken: 149.25s | ETC: 215.75s | Epoch Step: 101/450 | Loss: 0.5611 | Tokens per Sec: 2254.51 |
2023-11-03 17:43:20,856 - INFO - | Epoch: 1 | Total Progress: 48.22% | Training Time Taken: 178.20s | ETC: 190.80s | Epoch Step: 201/450 | Loss: 0.4377 | Tokens per Sec: 2096.31 |
2023-11-03 17:43:47,818 - INFO - | Epoch: 1 | Total Progress: 55.63% | Training Time Taken: 205.16s | ETC: 162.84s | Epoch Step: 301/450 | Loss: 0.5410 | Tokens per Sec: 2247.97 |
2023-11-03 17:44:15,021 - INFO - | Epoch: 1 | Total Progress: 63.04% | Training Time Taken: 232.37s | ETC: 135.63s | Epoch Step: 401/450 | Loss: 0.5047 | Tokens per Sec: 2224.86 |
```

**Comment by alexandreLamarre** [comment]
It also never updates status with being finished, to train new models, will have to uninstall with the workaround in https://github.com/rancher/opni/pull/1822

**Comment by alexandreLamarre** [comment]
training-controller logs:

```
2023-11-03 17:51:16,648 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:51:17,936 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:51:17,939 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:52:46,060 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:52:46,068 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:52:47,989 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:52:47,995 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:52:49,553 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:52:49,556 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:53:18,713 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:53:18,716 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:56:46,069 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:56:46,074 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:56:46,537 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:56:46,540 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:56:47,215 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:56:47,224 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:56:48,896 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:56:48,900 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:56:49,561 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:56:49,563 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:57:16,045 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:57:16,048 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:57:17,604 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:57:17,608 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:57:17,991 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:57:17,993 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:57:18,108 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:57:18,111 - INFO - received inferencing request. response : b'NO'
2023-11-03 17:59:46,957 - ERROR - Expecting value: line 2 column 1 (char 1)
2023-11-03 17:59:46,960 - INFO - received inferencing request. response : b'NO'
```

**Comment by alexandreLamarre** [comment]
Fixed in release 12.1
